spec/reek/smells/subclassed_from_core_class_spec.rb

```diff
@@ -0,0 +1,86 @@
+require_relative '../../spec_helper'
+require_lib 'reek/smells/subclassed_from_core_class'
+require_relative 'smell_detector_shared'
+
+RSpec.describe Reek::Smells::SubclassedFromCoreClass do
+  let(:detector) { described_class.new }
+
+  it_should_behave_like 'SmellDetector'
+
+  it 'should not report if the class has no ancestor' do
```

Comment:
Thats a little misleading dince it does have a superclass / ancestor. How about "does not inherit from a core class"?